contents/docs/feature-flags/early-access-feature-management.mdx

```diff
@@ -1,33 +1,112 @@
 ---
-title: Early Access Feature Management
+title: Early access feature management
 ---
 
-Early access feature management enables your users the ability to opt-in to betas and other early features. This is especially useful if you don't want to run a beta application process, or if you ship frequent updates and want users to control their own experience. 
+Early access feature management enables your users to opt in (and out) of betas and other in-progress features. 
 
-Users can also decide to opt-out of betas they do not want to be part of. PostHog tracks opt-ins and opt-outs, so you can see how popular a beta is with your users. 
+This is useful if you want to:
+- Run a beta program without building a custom solution
+- Provide access to features that are not yet ready for general release
+- Allow users to control their own product experience
 
 ## How it works
 
-Early access features can be configured in the early access menu. Features are powered by feature flags, so each early access feature must be linked to an existing feature flag or a feature flag with the equivalent name to the feature will be implicitly created and linked.
+Early access features can be created and edited from the "[Early Access Management](https://app.posthog.com/early_access_features)" tab in PostHog.
```

Comment:
Nit: dont need quotes since the text is already highlighted
```suggestion
Early access features can be created and edited from the [Early Access Management](https://app.posthog.com/early_access_features) tab in PostHog.
```